src/components/book-card/z-book-card/styles.css

```diff
@@ -0,0 +1,192 @@
+:host {
+  display: block;
+  max-width: fit-content;
+}
+
+:host > article {
+  background-color: var(--color-surface01);
+  color: var(--color-default-text);
+  font-family: var(--font-family-sans);
+}
+
+/* Tablet breakpoint */
+@media (min-width: 768px) {
+  :host > article.landscape {
+    width: 635px;
+    min-height: 332px;
+    max-height: 428px;
+    box-sizing: border-box;
+    border: solid var(--border-size-medium) var(--color-surface03);
+    border-radius: var(--border-radius);
+
+    .main-content {
+      display: flex;
+      height: 296px;
+      padding: calc(var(--space-unit) * 2);
+      gap: calc(var(--space-unit) * 2);
+
+      .cover {
+        display: flex;
+        overflow: hidden;
+        width: 221px;
+        border-radius: var(--border-radius);
```

Comment:
la mia impressione è che questo border-radius in realtà non abbia effetto sull'immagine. Provando a spostarlo sul tag img mi pare che i bordi, invece, si arrotondino.

Reply:
Si tratta di un effetto ottico dovuto alle dimensioni delle cover, suppongo. Se provi ad aprile il file pages/book-cards.html vedrai l'effetto diverso in base alla dimensione della copertina (cosa che, al solito, sul figma non viene mai considerata)